tests/macros/test-compat.js

```diff
@@ -0,0 +1,58 @@
+/* jshint node: true, mocha: true, esversion: 6 */
+
+const fs = require('fs'),
+      path = require('path'),
+      sinon = require('sinon'),
+      utils = require('./utils'),
+      chai = require('chai'),
+      chaiAsPromised = require('chai-as-promised'),
+      assert = chai.assert,
+      itMacro = utils.itMacro,
+      describeMacro = utils.describeMacro,
+      beforeEachMacro = utils.beforeEachMacro,
+      fixture_dir = path.resolve(__dirname, 'fixtures/compat');
+
+// Let's add "eventually" to assert so we can work with promises.
+chai.use(chaiAsPromised);
+
+// Load fixture data.
+const fixtures = {
+    alternative_name: {
+        data: null,
+        dataFile: 'alternative_name.json',
+        expected: '',
+        expectedFile: 'alternative_name-expected.txt',
+    }
+};
+
+for (const name in fixtures) {
+    fixtures[name].data = JSON.parse(
+        fs.readFileSync(path.resolve(fixture_dir, fixtures[name].dataFile), 'utf8')
+    );
+
+    fixtures[name].expected =
+        fs.readFileSync(path.resolve(fixture_dir, fixtures[name].expectedFile), 'utf8');
+}
+
+describeMacro('Compat', function () {
+
+    beforeEachMacro(function (macro) {
+        // insert magic here to make bcd fixtures available
```

Comment:
Here's what I would add here:
```
macro.ctx.require = sinon.stub();
macro.ctx.require.withArgs('mdn-browser-compat-data').returns(
    fixtures.alternative_name.data
);
```
But when I do that, it seems like the call to `require('mdn-browser-compat-data')` (which should call the stub) is not returning the fixture data. I need to look into that more on Monday.